server/controllers/deactivateTemporary/confirm.ts

```diff
@@ -0,0 +1,100 @@
+import { NextFunction, Response } from 'express'
+import FormWizard from 'hmpo-form-wizard'
+import backUrl from '../../utils/backUrl'
+
+export default class DeactivateTemporaryConfirm extends FormWizard.Controller {
+  middlewareSetup() {
+    super.middlewareSetup()
+    this.use(this.getResidentialSummary)
+  }
+
+  async getResidentialSummary(req: FormWizard.Request, res: Response, next: NextFunction) {
+    const { user, location } = res.locals
+    const { authService, locationsService } = req.services
+
+    const token = await authService.getSystemClientToken(user.username)
+    res.locals.residentialSummary = await locationsService.getResidentialSummary(token, location.prisonId)
+
+    next()
+  }
+
+  generateChangeSummary(cellWorkingCapacity: number, overallWorkingCapacity: number): string | null {
+    if (cellWorkingCapacity === 0) return null
+
+    const newOverallVal = overallWorkingCapacity - cellWorkingCapacity
+
+    return `\
+      You are making 1 cell inactive.
+      <br/><br/>
+      This will reduce the establishment’s total working capacity from ${overallWorkingCapacity} to ${newOverallVal}.
+    `.replace(/^\s*|\s*$/gm, '')
+  }
+
+  // eslint-disable-next-line no-underscore-dangle
+  async _locals(req: FormWizard.Request, res: Response, next: NextFunction) {
```

Comment:
How come you needed to override `_locals` as well as `locals`?

Reply:
I needed to override `_locals` because `locals` is not awaited so I can't convert it to `async` and use `await` inside of it. `_locals` is passed into the router and automatically awaited by express